Makefile

```
@@ -33,5 +33,5 @@ dist: clean
 	python setup.py bdist_wheel --universal
 
 release: dist
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)